stable/ambassador/templates/ambassador-pro-redis.yaml

```
@@ -0,0 +1,33 @@
+{{- if .Values.pro.enabled -}}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: ambassador-pro-redis
```

should add missing metadata and use template for names so its possible to install multiple releases, this file and others :) 

```suggestion
  name: {{ include "ambassador.fullname" . }}-pro-redis
  labels:
    app.kubernetes.io/name: {{ include "ambassador.name" . }}
    helm.sh/chart: {{ include "ambassador.chart" . }}
    app.kubernetes.io/instance: {{ .Release.Name }}
    app.kubernetes.io/managed-by: {{ .Release.Service }}
```

ditto on selectors.matchLabels and template labels

---

Make sure to get the matchLabels as well
```yaml
spec:
  replicas: 1
  selector:
    matchLabels:
      app.kubernetes.io/name: {{ include "ambassador.name" . }}
      app.kubernetes.io/instance: {{ .Release.Name }}
```

---

@Flydiverny 
```yaml
labels:
  app.kubernetes.io/name: {{ include "ambassador.name" . }} -pro-redis
```